src/definitions/h.json

```diff
@@ -0,0 +1,125 @@
+[
+  {  
+      "display": "Place the punctuation",
+      "description": "The hanging-punctuation property specifies whether a punctuation mark may be placed outside the line box at the start or at the end of a full line of text.",
+      "link": "https://www.w3schools.com/cssref/css3_pr_hanging-punctuation.asp",
+      "propertyName": "hanging-punctuation",
+      "valuetemplate": "{0}",
+      "categories": ["misc"],
+      "tags": "punctuation, character, paragraph",
+      "controls": [
+        {
+          "type": "select",
+          "default": "none",
+          "label": "Value",
+          "options": [
+            {
+              "name": "No Punctuation",
+              "value": "none"
+            },
+            {
+              "name": "At the First",
+              "value": "first"
+            },
+            {
+              "name": "At the Last",
+              "value": "last"
+            },
+            {
+              "name": "Allow at the End",
+              "value": "allow-end"
+            },
+            {
+              "name": "Force at the End",
+              "value": "force-end"
+            },
+            {
+              "name": "Default Value",
+              "value": "initial"
+            },
+            {
+              "name": "Inherit from parent",
+              "value": "inherit"
+            }
+          ]
+        }
+      ]
+    },
+    {  
+        "display": "Set the height",
+        "description": "The height property sets the height of an element. The height of an element does not include padding, borders, or margins!",
+        "link": "https://www.w3schools.com/cssref/pr_dim_height.asp",
+        "propertyName": "height",
+        "valuetemplate": "{0}",
+        "categories": ["misc", "layout", "container"],
+        "tags": "size, height, scale",
+        "controls": [
+          {
+            "type": "select",
+            "default": "auto",
+            "label": "Value",
+            "options": [
+              {
+                "name": "Auto",
+                "value": "auto"
+              },
+              {
+                "name": "Set in pixel(px)",
+                "value": "50px"
+              },
+              {
+                "name": "Set in percentage(%)",
```

Comment:
The default value should be derived from the actual attribute's default value which is "auto" for the "height" attribute.

Reply:
@koustov I didn't understand this one fully. I have given the default value as auto for height.